101-private-endpoint-webapp/azuredeploy.json

```diff
@@ -0,0 +1,224 @@
+            }
+        },
+        "SKU_name": {
+            "type": "String",
+            "defaultValue": "P1v2"
+        },
+        "SKU_tier": {
+            "type": "String",
+            "defaultValue": "PremiumV2"
+        },
+        "SKU_size": {
+            "type": "String",
+            "defaultValue": "P1v2"
+        },
+        "SKU_family": {
+            "type": "String",
+            "defaultValue": "P1v2"
+        },
+        "privateEndpoint_name": {
+            "type": "string"
+        },
+        "privateLinkConnection_name": {
+            "type": "string"
+        },
+        "privateDNSZone_name": {
+            "type": "string",
+            "defaultValue": "privatelink.azurewebsites.net"
+        }                       
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2020-04-01",
+            "name": "[parameters('virtualNetwork_name')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[parameters('virtualNetwork_CIDR')]"
+                    ]
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Network/virtualNetworks/subnets",
+            "apiVersion": "2020-04-01",
+            "name": "[concat(parameters('virtualNetwork_name'),'/', parameters('subnet1_name'))]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetwork_name'))]"
+            ],
+            "properties": {
+                "addressPrefix": "[parameters('subnet1_CIDR')]",
+                "privateEndpointNetworkPolicies": "Disabled"
+            }
+        },
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2019-08-01",
+            "name": "[parameters('serverFarm_name')]",
+            "location": "[parameters('location')]",
+            "sku": {
+                "name": "[parameters('SKU_name')]",
+                "tier": "[parameters('SKU_tier')]",
+                "size": "[parameters('SKU_size')]",
+                "family": "[parameters('SKU_family')]",
+                "capacity": 1
+            },
+            "kind": "app"
+
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2019-08-01",
+            "name": "[parameters('site_name')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', parameters('serverFarm_name'))]"
+            ],
+            "kind": "app",
+            "properties": {
+                "enabled": true,
+                "hostNameSslStates": [
+                    {
+                        "name": "[concat(parameters('site_name'), '.azurewebsites.net')]",
```

Comment:
remove the hardcoded "azurewebsites.net"

Reply:
done